test/integration/api_docs/services_controller_test.rb

```diff
@@ -43,7 +43,7 @@ def test_index_and_show
       assert_response :success
       assert_equal ['host', 'apis'], index_result.keys
 
-      api_expected_names = ['Service Management API', 'Account Management API', 'Analytics API', 'Billing API']
+      api_expected_names = ['Service Management API', 'Account Management API', 'Analytics API', 'Billing API', 'Policy Registry API']
```

Comment:
Shouldn't we use here the computed values in config/locales/en.yml? To detect typos and in case value change, etc.

Reply:
Yes, we should 👍

Reply:
Sorry, I thought it was about the implementation, not the test. The implementation is already consistent with the locale: https://github.com/3scale/porta/blob/09b2ca44526f11b3894799a6e32f859766ce444c/app/controllers/api_docs/services_controller.rb#L101

The test is OK since we want to assert that those are the API names shown in the index page.